codegenerator/cli/templates/static/codegen/src/db/DbFunctions.res

```diff
@@ -38,7 +38,7 @@ let makeTableBatchSet = (table, schema: S.t<'entity>) => {
 
     let unsafeSql =
       `
-INSERT INTO "public".${table.tableName->quote} (${quotedFieldNames->Js.Array2.joinWith(", ")})
+INSERT INTO ${Env.Db.publicSchema}.${table.tableName->quote} (${quotedFieldNames->Js.Array2.joinWith(", ")})
```

Comment:
With unsafe interpolation this needs to be quoted to handle schema names with mixed case
```suggestion
INSERT INTO ${Env.Db.publicSchema->quote}.${table.tableName->quote} (${quotedFieldNames->Js.Array2.joinWith(", ")})
```